src/tools/icn2img.cpp

```diff
@@ -102,17 +105,13 @@ int main( int argc, char ** argv )
             dstfile += ".png";
             shortdstfile += ".png";
 #endif
-            if ( fheroes2::Save( image, dstfile, 23 ) ) {
-                fs << " <sprite index=\"" << ii + 1 << "\" name=\"" << shortdstfile.c_str() << "\" ox=\"" << head.offsetX << "\" oy=\"" << head.offsetY << "\"/>"
-                   << std::endl;
-            }
+            std::cout << "Image " << ii + 1 << " has offset of [" << head.offsetX << ", " << head.offsetY << "]" << std::endl;
```

Comment:
```suggestion
            std::cout << "Image " << ii + 1 << " has offset of [" << static_cast<int32_t>( head.offsetX ) << ", " << static_cast<int32_t>( head.offsetY ) << "]" << std::endl;
```
Otherwise negative offsets look really funny (-1 becomes 65535 and so on) ;)